### DISCUSSION TITLE
Cannot mark complete the assignment

### DISCUSSION THREAD

**Discussion by AndyKhang404** [skills/.github#387] [Code with Copilot]
I'm able to follow all the steps without any issue. However, when I click "mark complete", it says that I only score 5 out of 6 points. Now I have no idea where i've done wrong. Please help me :(
![Screenshot 2023-09-23 184142](https://github.com/skills/.github/assets/75353820/1079c377-c837-4449-ab01-ed5445ee65dc)
![Screenshot 2023-09-23 184219](https://github.com/skills/.github/assets/75353820/3cd69ea9-8ef7-419a-9936-510090b7b117)

**Comment by cmwilson21** [comment]
@AndyKhang404 It looks like this may be a course taken through GitHub's Education program 📚 

Please open a discussion [here](https://github.com/orgs/community/discussions/categories/github-education) describing your issues 👍

**Reply by AndyKhang404** [reply]
Thanks for replying

**Comment by vitamin2025** [comment]
Hello, I faced the same problem, and it seems like one of the bot's numbering problems. But I got a solution for this!

So, basically the bot had to declare that we have finished step 5, instead it declared that we have finished "step X". As you can see below:

![image](https://github.com/skills/.github/assets/72799488/43b88b65-7612-4e5b-a153-68dcf4ab3d05)

**The Solution for this is** 

Step 1. Go to your repo and open -step.txt folder.
You can locate this folder in .github > steps 

Step 2. Change X to 5 as shown below:

![image](https://github.com/skills/.github/assets/72799488/76b59128-d1a3-42b6-a1b3-bb5207fd3a00)


Step 3. Now push it and wait for 1 min, it will fix the problem!
